api/src/routing.js

```diff
@@ -530,6 +530,21 @@ const writeParsedBody = (proxyReq, req) => {
   }
 };
 
+const copyProxyHeaders = (proxyRes, res) => {
```

Comment:
Putting this in a function, cause I'll call it for the intercepted audit proxy for restricted users as well.